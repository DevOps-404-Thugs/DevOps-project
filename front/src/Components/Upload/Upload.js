import React, { useEffect, useState } from 'react'

//取参参考这个代码哈
class Upload extends React.Component {
    constructor(props){
        super(props);
        this.state = {
            id: this.props.match.params.id
        };
		}
		
    render(){
			return(
        <div>
          <p>{this.state.id}</p>
        </div>
      );
    }
}
export default Upload;