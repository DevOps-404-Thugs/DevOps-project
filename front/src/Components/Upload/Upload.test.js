import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Upload from './Upload';


describe('Upload component', () =>{
    test("match the snapshot", () =>{
        const component = renderer.create(
            <BrowserRouter>
                <Upload/>
            </BrowserRouter>
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });  
});