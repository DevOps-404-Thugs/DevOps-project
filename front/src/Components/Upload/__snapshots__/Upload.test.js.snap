// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload component match the snapshot 1`] = `
<div>
  <div
    style={
      Object {
        "margin": "3rem auto",
        "width": "75%",
      }
    }
  >
    <form>
      <div
        class="form-group"
      >
        <label>
          House Name
        </label>
        <input
          class="form-control"
          id="email-input"
          onChange={[Function]}
          placeholder="Input your email address here"
          type="text"
        />
      </div>
      <div
        class="form-group"
      >
        <label>
          Address
        </label>
        <input
          class="form-control"
          id="password-input"
          onChange={[Function]}
          placeholder="Input your password here"
          type="text"
        />
      </div>
      <button
        class="btn btn-primary"
        onClick={[Function]}
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
